portal/src/ThemePresetWidget.tsx

```diff
@@ -161,7 +164,7 @@ function ThemePreset(props: ThemePresetProps) {
   return (
     <div
       className={cn(styles.preset, isSelected && styles.selected)}
-      onClick={onClick}
+      onClick={disabled ? undefined : onClick}
```

Comment:
Better to update the css `cursor` not to use `pointer` when it is disabled, the default it `auto`.

Reply:
Add disabled state.